packages/content/src/MediaGallery/index.js

```diff
@@ -137,7 +135,11 @@ const ActiveThumbnailButton = styled.button`
     outline: 2px solid #000;
   }
 
-  border: 1px solid #017acd;
+  & img {
+    width: 84px;
```

Comment:
Do we want to potentially stretch images this way?  What if we set a specific height (per the line below), but then just set a max-width?  Or perhaps ignored width altogether?

Reply:
We are already manipulating the size of the image in the other thumbanils in this same `ul`. See the `ThumbnailButton` style just above this `ActiveThumbnail`

Reply:
Gotcha.  Good call.